mne/viz/raw.py

```diff
@@ -183,6 +179,17 @@ def plot_raw(raw, events=None, duration=10.0, start=0.0, n_channels=20,
         Individual projectors can be enabled/disabled interactively (see
         Notes). This argument only affects the plot; use ``raw.apply_proj()``
         to modify the data stored in the Raw object.
+    group_by : str
+        How to group channels. 'type' groups by channel type, 'original'
+        plots in the order of ch_names, 'selection' uses Elekta's channel
+        groupings (only works for Neuromag data), 'position' groups the
+        channels by the positions of the sensors. 'selection' and 'position'
+        modes allow custom selections by using lasso selector on the topomap.
+        Pressing ``ctrl`` key while selecting allows appending to the current
+        selection. Channels marked as bad appear with red edges on the topomap.
+        'type' and 'original' groups the channels by type in butterfly mode.
```

Comment:
There is no mention of how this interacts with `order`. Which takes precedence if both are given?

You should probably mention `group_by` in the `order` description and vice-versa.